Multisig/UI/Assets/AssetsView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AssetsView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 13.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct AssetsView: View {
+    @ObservedObject
+    var model = AssetsViewModel()
+
+    var body: some View {
+        ZStack(alignment: .center) {
+            if model.isLoading {
```

Comment:
please create a new issue with a `technical debt` to refactor this piece into a separate more generic view that has loading, error, and content.

Reply:
As we still have time, why to make technical debt from now

Reply:
I think this should be done in another PR, to reduce growth of this PR

Reply:
done: https://app.zenhub.com/workspaces/safe-multisig---mobile-5c1b98250e13551e8aae81eb/issues/gnosis/safe-ios/128